src/main/java/uk/gov/companieshouse/api/accounts/CompanyAccountsApplication.java

```diff
@@ -58,13 +63,20 @@ public ObjectMapper objectMapper() {
 
     @Override
     public void addInterceptors(final InterceptorRegistry registry) {
-        registry.addInterceptor(transactionInterceptor)
-                .addPathPatterns("/transactions/{transactionId}/**");
+        registry.addInterceptor(openTransactionInterceptor).order(Ordered.HIGHEST_PRECEDENCE)
+            .addPathPatterns("/transactions/{transactionId}/**");
+
+        registry.addInterceptor(closedTransactionInterceptor).order(Ordered.HIGHEST_PRECEDENCE)
+            .addPathPatterns(
+                "/private/transactions/{transactionId}/company-accounts/{companyAccountId}/filings");
+
         registry.addInterceptor(companyAccountInterceptor)
-                .addPathPatterns(
-                        "/transactions/{transactionId}/company-accounts/{companyAccountId}**");
+            .addPathPatterns(
+                "/transactions/{transactionId}/company-accounts/{companyAccountId}/**",
+                "/private/transactions/{transactionId}/company-accounts/{companyAccountId}/**");
```

Comment:
same as above, there shouldn't be a `/` before the `**` i.e.
`"/private/transactions/{transactionId}/company-accounts/{companyAccountId}/**"`